javascript/javascript2/week3/fastestPresser.js

```diff
@@ -0,0 +1,58 @@
+const timerInput = document.querySelector("#inputButton");
+const startGame = document.querySelector("#button1");
+const playerS = document.querySelector(".PlyS");
+const playerL = document.querySelector(".PlyL");
+const result = document.querySelector(".Result");
+const startAgain = document.querySelector(".Restart");
+
+let counterS = 0;
+let counterL = 0;
+
+//#2 Set a timeout for the time specified by the user:
+
+function playGame() {
+  setTimeout(() => {
```

Comment:
What if i press start when multiple times?

you should disable the the button on start `document.querySelector('#button1').disabled = true` that will prevent multiple games being started